.release-please-manifest.json

```diff
@@ -0,0 +1 @@
+{".": "2.3.0"}
```

Comment:
Will it both update the package.json and this manifest file? 🤔

I think it would only do one.

Reply:
Yeah the manifest is updated with the version bump, I don't really see why it is needed but without it the action step fails